packages/shared/src/components/widgets/SocialShare.tsx

```diff
@@ -21,78 +21,82 @@ import TelegramIcon from '../icons/Telegram';
 import { FeedItemPosition, postAnalyticsEvent } from '../../lib/feed';
 import { Origin } from '../../lib/analytics';
 import AnalyticsContext from '../../contexts/AnalyticsContext';
+import { Comment } from '../../graphql/comments';
 
 interface SocialShareProps {
   origin: Origin;
   post: Post;
+  comment?: Comment;
 }
 export const SocialShare = ({
   post,
+  comment,
   origin,
   columns,
   column,
   row,
 }: SocialShareProps & FeedItemPosition): ReactElement => {
+  const isComment = !!comment;
+  const link = isComment
+    ? `${post?.commentsPermalink}#c-${comment.id}`
```

Comment:
I think you should create a function that hashes the comment. I see you use it many times `c-${id}`. It's better to store it somewhere